entrypoint.sh

```diff
@@ -8,22 +9,79 @@ if [ ! -e $image_path ]; then
   if [ -e $zip_path ]; then
       echo "Extracting fresh filesystem..."
       unzip $zip_path
-      mv *.img $image_path
+      mv -- *.img $image_path
```

Comment:
What do these dashes do?

Reply:
https://unix.stackexchange.com/a/11382

Reply:
I ran it through `shellcheck` and this is a suggestion. I've started doing it to most of my code so I think this was picked up in a linting spree.